code/modules/events/electrical_storm.dm

```diff
@@ -5,6 +5,21 @@
 	has_skybox_image = TRUE
 	var/list/valid_apcs		// Shuffled list of valid APCs.
 	var/global/lightning_color
+	var/last_bounce //track the last time we bounced around whatever overmap effect we're affecting.
+	var/bounce_delay = 10 SECONDS // this is divided by severity, so a intense storm means you're going *flying*.
+	var/static/lightning_sounds = list(
```

Comment:
Lohi got grumpy about me suggesting this and said not to move it. :(

Reply:
This is the way I did it in the first place actually, his comments were on the medibot PR.

Reply:
Oh derp.